### PR TITLE
Added ICCN chunk index and parsed through document to add references to ICCN

### PR DESCRIPTION
**Pull request by digitaltvguy**
Updated index.html to allow merge (edited wrong version so had to back up and re-submit)

Added ICCN chunk index and parsed through document to add references to ICCN.

A few fixes to cICP.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6713,9 +6792,9 @@ <h2>Security
 "#ztxt-compressed-textual-data"><span class="chunk">zTXt</span></a> chunks contain keywords
 and data
 that are meant to be displayed as plain text. The <a href=
-"#iccp-embedded-icc-profile"><span class="chunk">iCCP</span></a> and <a href=
-"#splt-suggested-palette"><span class="chunk">sPLT</span></a> chunks contain
-keywords that are meant to be displayed as plain text. It is
+"#iccp-embedded-icc-profile"><span class="chunk">iCCP</span></a>, "#iccn-embedded-icc-profile">
```
Comment (ProgramMax) [inline index.html:6795]: This seems to be missing < a href=

File: index.html [hunk 2]
```diff
@@ -7209,10 +7288,10 @@ <h2>Decoder gamma
...
 <p>When the incoming image has unknown gamma (<a href=
 "#gama-image-gamma"><span class="chunk">gAMA</span></a>, <a href=
-"#srgb-standard-colour-space"><span class="chunk">sRGB</span></a>, and <a href=
-"#iccp-embedded-icc-profile"><span class="chunk">iCCP</span></a> all absent),
-standalone image viewers should
-choose
+"#srgb-standard-colour-space"><span class="chunk">sRGB</span></a>, <a href=
+"#iccp-embedded-icc-profile"><span class="chunk">iCCP</span></a>, and 
+<a href="#iccp-embedded-icc-profile"><span class="chunk">iCCP</span></a> 
+all absent), standalone image viewers should choose
```
Comment (ProgramMax) [inline index.html:7294]: This mentions iCCP twice. It is probably supposed to be iCCN.

**Review by ProgramMax — CHANGES_REQUESTED** [review]
A few changes need to be made.
Also, there are links to the fragment #iccn-embedded-icc-profile but no element has an ID of iccn-embedded-icc-profile.

**Comment by ProgramMax** [comment]
I should mention: Because I updated your pull request, you might need to $git pull before you make your changes.

File: index.html [hunk 3]
```diff
@@ -4481,7 +4502,61 @@ <h2><span class="chunk">cICP</span>
...
+    
+    <tr>
+    <td class="Regular">Compressed profile</td>
+    <td class="Regular">n bytes</td>
+    </tr>
+    </table>
+    
+    <p>The iCCN profile name may be any convenient name for referring to
+    the profile. It is case-sensitive. Profile names shall contain printable 
+    Latin-1 characters, spaces and includes UTF-8 characters.  Leading, trailing,
```
[the remark this answers is not in the document]
Comment (svgeesus) [reply index.html:4547]: Yes, just say "printable characters"

File: index.html [hunk 4]
```diff
@@ -4481,7 +4502,61 @@ <h2><span class="chunk">cICP</span>
 [[ITU-T H.273]] and
 it's associated recommendations.</p>
 </section>
-</section>
+
+<!-- Maintain a fragment named "11iCCN" to preserve incoming links to it -->
+<section id="11iCCN">
+  <h2><span class="chunk">iCCN</span>
+    Embedded ICC Profiles v4 and UTF-8 Profile Names</h2>
```
Comment (svgeesus) [inline index.html:4509]: Change "Embedded ICC Profiles v4 and UTF-8 Profile Names" to "Embedded ICC.2 Profiles [[ICC-2]][[ISO 20677-1]] and UTF-8 Profile Names"

File: index.html [hunk 5]
```diff
@@ -4481,7 +4502,61 @@ <h2><span class="chunk">cICP</span>
...
+    <td class="Regular">n bytes</td>
+    </tr>
+    </table>
+    
+    <p>The iCCN profile name may be any convenient name for referring to
+    the profile. It is case-sensitive. Profile names shall contain printable 
+    Latin-1 characters, spaces and includes UTF-8 characters.  Leading, trailing,
+    and consecutive spaces are not permitted.</p>
+  
+    <p>PNG files should not simultaneously contain embedded iCCP and iCCN tags.</p>
```
Comment (svgeesus) [inline index.html:4550]: Is that still true? It is now possible to put an ICC.2 (iccMax) profile into iCCN and an older ICC.1 profile into iCCP. So I suggest dropping this paragraph.

File: index.html [hunk 6]
```diff
@@ -1820,9 +1820,11 @@ <h2>Chunk types</h2>
   <li>Colour space information: <a href="#chrm-primary-chromaticities-and-white-point"><span class=
   "chunk">cHRM</span></a>, <a href="#gama-image-gamma"><span class=
   "chunk">gAMA</span></a>, <a href="#iccp-embedded-icc-profile"><span class=
-  "chunk">iCCP</span></a>, <a href="#sbit-significant-bits"><span class=
+  "chunk">iCCP</span></a>, <a href="#iccn-embedded-icc-profile"><span class=
+  "chunk">iCCN</span></a>, <a href="#sbit-significant-bits"><span class=
```
Comment (svgeesus) [inline index.html:1824]: As discussed on the call today, all instances of `iCCN` should now be changed to `iCCM` and similarly, to not break links, change all "iccn-embedded-icc-profile" to "iccm-embedded-icc-profile"

**Pull request closed without merging**